index.html

```diff
@@ -215,10 +215,10 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
         </p>
         <p>[[rfc7515]] MAY be used to secure this media type.</p>
```

Comment:
```suggestion
        <p>[[rfc7515]] MAY be used to secure this media type. The result of such securing MUST be used as the JWT component of an SD-JWT.</p>
```

Reply:
I agree with the intention, but I prefer to make these changes in a separate PR (That will be easier to review).